### PR TITLE
Added example config for Ender5Pro 422 with Micro-Swiss Direct Drive and BLtouch 

### PR DESCRIPTION
**Pull request by JMLdev**
There is currently no example configuration for Ender 5 Pro with BL touch. It took quite some time to figure out all of these settings and I would like to save someone else the effort. There are a few tweaks in my config files, but they're explained in the README along with instructions to change them back to stock. These are the values that I arrived at after running through two days of intense testing. This build has been tested and confirmed with my Ender 5 Pro with BLtouch and Micro-Swiss direct drive.

### CODE REVIEW

**Comment by JMLdev** [comment]
This also has long filename support enabled, and a setting to disable software endstops for calibration of the BL touch probe.

**Comment by JMLdev** [comment]
This has a few advanced menus added to the default interface for convenience.

I removed double probing because it's annoying and takes too long.

File: config/examples/Creality/Ender-5 Pro/CrealityV422 BLtouch Micro-Swiss Direct Drive/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Configuration files for Stock Ender 5 Pro 4.2.2 Board
+
+This configuration has some tweaks to jerk and acceleration. If you don't want those changes then set these values in the `Configuration.h` file for the following commands.
+
+- Line 1062: `#define DEFAULT_XJERK 10.0`
```
Comment (thisiskeithb) [inline config/examples/Creality/Ender-5 Pro/CrealityV422 BLtouch Micro-Swiss Direct Drive/README.md]: Line numbers will change, so it’s best to not reference them directly.

Comment (JMLdev) [reply config/examples/Creality/Ender-5 Pro/CrealityV422 BLtouch Micro-Swiss Direct Drive/README.md]: Good point. I removed those references, updated some instructions for clarity, and added the boot screen and status screen files.

**Comment by JMLdev** [comment]
@thisiskeithb I addressed your recommendation. Please resolve conversation. Thanks.

**Comment by thisiskeithb** [comment]
> I addressed your recommendation. Please resolve conversation. Thanks.

Only thinkyhead can resolve conversations and he'll take care of it when he does a final pass before merging.

**Comment by JMLdev** [comment]
> > I addressed your recommendation. Please resolve conversation. Thanks.
> 
> Only thinkyhead can resolve conversations and he'll take care of it when he does a final pass before merging.

Okay cool. Thanks for the review.

**Comment by JMLdev** [comment]
Lowered jerk to a more reasonable amount based on further testing and added HOST_ACTION_COMMANDS for OktoPrint support.

**Comment by thinkyhead** [comment]
Instead of making a separate configuration for all this particular combination of add-on hardware, I've opted here to just add a couple of conditionals and apply the other enhancements to the existing `CrealityV422` config. I reverted the E-steps to 93 because a hotend alone should not be changing that number.

**Comment by JMLdev** [comment]
> Instead of making a separate configuration for all this particular combination of add-on hardware, I've opted here to just add a couple of conditionals and apply the other enhancements to the existing `CrealityV422` config. I reverted the E-steps to 93 because a hotend alone should not be changing that number.

It isn't just a hotend, it's the direct drive conversion, which changes the gear ratio, and therefore the esteps.

**Comment by JMLdev** [comment]
Is anyone going to complete this pull request or request revisions?

**Comment by thisiskeithb** [comment]
> It isn't just a hotend, it's the direct drive conversion, which changes the gear ratio, and therefore the esteps.

Since this has been merged, you'll need to submit a new PR with this change. Just wrap it in `#if ENABLED(ENDER5_USE_MICROSWISS)` and use an `#else` for the old values.

**Comment by thisiskeithb** [comment]
> It isn't just a hotend, it's the direct drive conversion, which changes the gear ratio, and therefore the esteps.

I've submitted https://github.com/MarlinFirmware/Configurations/pull/790 to take care of this.

**Pull request merged**